File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    var answer = 0;
    
    for (let b of babbling) {
        for (let tmp of ["aya", "ye", "woo", "ma"]) {
            b = b.replace(tmp, "0")
        }
    
        if (b.split("").every(i => i === "0")) answer += 1
    }
    
    
    return answer;
}

File: 프로그래머스/unrated/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    answer = 0
    temp_set = {"aya", "ye", "woo", "ma"}  # set으로 변경
    tmp = []
    
    for b in babbling:
        for t in temp_set:
            if t in b:
                b = b.replace(t, " ")  # replace 메서드가 반환값을 가지므로 b에 할당
                
                if (b == " " or b == "  " or b == "   " or b == "    "):
                    tmp.append(b)
    
    return len(tmp)
